src/realty/domain/realty.spec.ts

```diff
@@ -358,4 +358,63 @@ describe('Realty', () => {
       });
     });
   });
+
+  describe('when remove image', () => {
+    let realty: Realty;
+    const images: RealtyImage[] = [
+      new RealtyImage({
+        id: 1,
+        url: 'https://myimage.com/1',
+        isCover: true,
+      }),
+      new RealtyImage({
+        id: 2,
+        url: 'https://myimage.com/2',
+        isCover: false,
+      }),
+    ];
+
+    beforeEach(() => {
+      const expected_value = {
+        title: 'my realty',
+        price: 750000,
+        type: 'HOUSE',
+        images,
+      };
+      realty = new Realty({
+        title: expected_value.title,
+        price: expected_value.price,
+        type: RealtyType.APARTMENT,
+        images: expected_value.images,
+      });
+    });
+
+    it('returns image list without removed image', () => {
+      const imageId = 2;
+      realty.removeImage(imageId);
+      expect(realty.images).toHaveLength(1);
+      expect(realty.images).toEqual([
+        new RealtyImage({
+          id: 1,
+          url: 'https://myimage.com/1',
+          isCover: true,
+        }),
+      ]);
+    });
+
+    it('return an empty realty images', () => {
```

Comment:
Senti falta dos describes descrevendo o cenário
```suggestion
    describe('when its last image in the list', () => { 
      it('return an empty realty images', () => {
```